opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -125,8 +149,50 @@
         /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
         SpanBuilder withStartTimestamp(long microseconds);
 
-        /** Returns the started Span. */
-        Span start();
+        /**
+         * Returns a newly started and activated {@link ActiveSpan}.
+         *
+         * <p>
+         * The returned {@link ActiveSpan} supports try-with-resources. For example:
+         * <pre>{@code
+         *     try (ActiveSpan span = tracer.buildSpan("...").startActive()) {
+         *         // (Do work)
+         *         span.setTag( ... );  // etc, etc
+         *     }  // Span finishes automatically unless pinned via {@link ActiveSpan#capture }
+         * }</pre>
+         *
+         * <p>
+         * If
```

Comment:
This text is mentioned in `addReference` and `startActive` but not in `startManual` - might cause confusion about whether the implement CHILD_OF relationship is setup in the manual case, even though it is made clear elsewhere.